packages/backend/src/peripherals/database/RpcTransactionCountRepository.ts

```diff
@@ -90,9 +91,7 @@ export class RpcTransactionCountRepository extends BaseRepository {
     noPrevBlockNumbers.push(Infinity)
     noNextBlockNumbers.unshift(-Infinity)
 
-    if (noNextBlockNumbers.length !== noPrevBlockNumbers.length) {
-      throw new Error('Arrays length should be the same')
-    }
+    assert(noNextBlockNumbers.length === noPrevBlockNumbers.length)
```

Comment:
This assert forgets the message

Reply:
I removed it because it's obiovus. IMO message should explain some deeper reasoning, but it was just explaining what was being checked.